store/models/log_events.go

```diff
@@ -347,29 +374,26 @@ func parseRunLog20190123(log Log) (JSON, error) {
 	return js.Add("functionSelector", OracleFulfillmentFunctionID20190123)
 }
 
-func parseRunLog20190128(log Log) (JSON, error) {
+func parseRunLog20190207(log Log) (JSON, error) {
```

Comment:
These versions are a bit hard to distinguish, could we add a semantic description or keyword of some sort? e.g: `20190207_sql_release`

Reply:
Added some documentation to each method to indicate how it is different from the previous version.